### DISCUSSION TITLE
Хроники Хаоса бот

### DISCUSSION THREAD

**Discussion by GameDev111333** [eigenein/bestmobabot#32] [Q&A]
Приветствую.
Есть ли возможность написать бота, который будет сам бить Повелителей в Турнире Стихий (цеплять seed с сервера) и набивать результат в атаке, который я уже когда-либо набирал?
Есть заготовки и визуально как это уже происходит, но у кого есть код не делятся им, можешь в этом помочь?
Напиши контакты для связи: телеграм, вк, дискорд?

**Comment by eigenein** [comment]
Привет! Я больше не занимаюсь ботом и не играю в «Хроники», увы
